newrelic_lambda_cli/integrations.py

```diff
@@ -590,7 +603,9 @@ def auto_install_license_key(input):
 @catch_boto_errors
 def install_license_key(input, nr_license_key, policy_name=None, mode="CREATE"):
     assert isinstance(input, (IntegrationInstall, IntegrationUpdate))
-    lk_stack_status = _get_cf_stack_status(input.session, LICENSE_KEY_STACK_NAME)
+    lk_stack_status = _get_cf_stack_status(
+        input.session, LICENSE_KEY_STACK_NAME, input.nr_account_id
+    )
```

Comment:
A user who has an existing secret with no outputs (because they used an older CLI) will show up as having the stack, and will see the warning, right? We should probably think about how to avoid having them see the warning forever.

Reply:
Not totally following you here. Also, Is that something we want to solve in this ticket?

Reply:
This is an upgrade scenario: user has an existing, working install, and they do an upgrade. They should not get a warning message. So the test case would look like:

1. Install a region with CLI version 0.5.5.
2. Run an upgrade with CLI version <this version>
3. Ensure that this happy-path upgrade doesn't generate a warning message.

Reply:
Also, speaking of versions, we should change the version in setup.py to 0.6.0 or something :)

Reply:
Ahh yes, I'm tracking now.